gravity-forms/gw-zip-files.php

```diff
@@ -315,8 +315,12 @@ public function get_zip_paths( $entry, $type = false ) {
 		return $type ? rgar( $paths, $type ) : $paths;
 	}
 
-	public function get_zip_filename( $entry_id ) {
-		return $this->get_slug( $this->_args['zip_name'], $entry_id, $this->_args['field_ids'] ) . '.zip';
+	public function get_zip_filename( $entry ) {
+		$form_id  = $entry['form_id'];
+		$form     = GFAPI::get_form( $form_id );
+		// replace merge tags in the zip file name
+		$zip_name = GFCommon::replace_variables( $this->_args['zip_name'], $form, $entry, false, false, false );
```

Comment:
```suggestion
		$zip_name = GFCommon::replace_variables( $this->_args['zip_name'], $form, $entry, false, false, false, 'text' );
```

I'd recommend changing this to `text` to HTML isn't added to the filename for some merge tags.